src/components/Hero.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import '../index.css';
+
+const Hero = ({ title, subtitle, buttonText, onClick, heroImage }) => {
+  const heroBackground = {
+    backgroundImage: `url(${heroImage})`
+  };
+  return (
+    <div className="hero-container" style={heroBackground}>
+      <div className="hero-content">
+        <h1>{title}</h1>
+        <h3>{subtitle}</h3>
```

Comment:
Best practice is to not skip levels of headings. If you stick to the levels it makes it easier for screen reader users to follow along on the page. If you change the h3 to a h2 you would follow the levels of headings. If you want the h2 to be the same size as the h3 you can style it in CSS.